config/locales/en.yml

```diff
@@ -32,12 +32,12 @@
 en:
   # start documentation links
   docs:
-    base_url: 'https://access.redhat.com/documentation/en-us/%{docs_version}/html'
+    base_url: 'https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/%{docs_version}/html'
```

Comment:
this is same in saas and on-prem now, right?

Reply:
Yes!!! Finally